app/models/page.rb

```diff
@@ -27,9 +27,9 @@ def use_slug
     @use_slug || true
   end
 
-  # TODO What's this for?
+  # TODO What"s this for?
```

Comment:
Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.